packages/expo-module-template/example/babel.config.js

```diff
@@ -1,19 +1,6 @@
-const path = require('path');
 module.exports = function (api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
-    plugins: [
-      [
-        'module-resolver',
```

Comment:
We removed this dependency from `babel-preset-expo` so it may no longer be shipped in projects that are using it indirectly. While `extraNodeModules` is a bit of hack (it checks for existing modules before applying the alias), it is faster/safer to use resolver changes (aliases) than it is to use transform changes (babel) as the caching is simpler.

Reply:
Yeah, thanks for the explanation!